packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1505,6 +1505,9 @@ export class GarbageCollector implements IGarbageCollector {
         // For summarizer client, queue the event so it is logged the next time GC runs if the event is still valid.
         // For non-summarizer client, log the event now since GC won't run on it. This may result in false positives
         // but it's a good signal nonetheless and we can consume it with a grain of salt.
+        // Events generated:
+        // InactiveObject_Loaded, InactiveObject_Changed, InactiveObject_Revived
+        // SweepReadyObject_Loaded, SweepReadyObject_Changed, SweepReadyObject_Revived
```

Comment:
Maybe mention that these are more severe than the Inactive ones?  @agarwal-navin do we maybe want to downgrade the InactiveObject ones to generic? I think so because they're not immediately actionable.  SweepReady ones are moreso IMO so can stay error

Reply:
I've updated the comments to talk about severity

Reply:
Do we want to downgrade the inactiveObject errors in a separate PR? Let's not block the comments.

Reply:
I would say let it remain the way it is. If we want to downgrade inactive object error, let's create and issue and discuss it there.